tracer/build/artifacts/dd-dotnet.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+# Get the directory of the script
+DIR=$(dirname "$0")
+
+
+# Read the /etc/os-release to detect distribution
+if [ -f /etc/os-release ]; then
+    . /etc/os-release
+else
+    echo "Unable to find /etc/os-release. Cannot determine the distribution."
+    exit 1
+fi
+
+# Check the machine architecture
+ARCH="$(uname -m)"
+
+DD_DOTNET_PATH=""
+
+# Set the DD_DOTNET_PATH according to the distribution and architecture
+if [ "$ID" = "alpine" ]; then
```

Comment:
where is `$ID` set? 🤔

Reply:
It's pure magic.
The content of /etc/os-release if formatted as variables:
```
PRETTY_NAME="Ubuntu 22.04.1 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.1 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
```
So when sourcing it with `. /etc/os-release`, the values are available as variables in the bash script.

Reply:
🤯